truss/test_data/model_load_failure_test/config.yaml

```diff
@@ -0,0 +1,36 @@
+apply_library_patches: true
+bundled_packages_dir: packages
+data_dir: data
+description: null
+environment_variables: {}
+examples_filename: examples.yaml
+external_package_dirs: []
+input_type: Any
+live_reload: false
+model_class_filename: model.py
+model_class_name: Model
+model_framework: custom
+model_metadata: {}
+model_module_dir: model
+model_name: null
+model_type: custom
+python_version: py39
+requirements: []
+resources:
+  accelerator: null
+  cpu: 500m
+  memory: 512Mi
+  use_gpu: false
+secrets: {}
+spec_version: '2.0'
+system_packages: []
+train:
```

Comment:
you can drop the train config or any of the other keys that you don't care about here. Let's try to make this minimal